src/storage/src/hummock/store/version.rs

```diff
@@ -427,6 +424,7 @@ impl HummockVersionReader {
                 return Ok(data.into_user_value());
             }
         }
+        local_stats.found_key = true;
```

Comment:
IMO, setting the value directly is prone to some omissions, which leads to errors in reporting

How about using some intuitive functions like:
- report_for_get_found_key
- report_for_get_not_found_key